File: UNIDAD 1/EXAMEN_EU3_GR10/VIAJECITOS_REST_DOTNET_GR10/02. CLIWEB/VIAJECITOS_REST_CLIWEB_GR10/VIAJECITOS_REST_CLIWEB_GR10/Models/Factura.cs
using ViajecitosRestServer.Models;

namespace VIAJECITOS_REST_CLIWEB_GR10.Models
{
    public class Factura
    {
        public int IdFactura { get; set; }
        public string NumeroFactura { get; set; }
        public DateTime FechaEmision { get; set; }
        public int IdEmpleado { get; set; }
        public int IdCliente { get; set; }
        public int IdMetodoPago { get; set; }
        public decimal Subtotal { get; set; }
        public decimal Descuento { get; set; }
        public decimal Iva { get; set; }
        public decimal Total { get; set; }
        public Empleado Empleado { get; set; }
        public Cliente Cliente { get; set; }
        public MetodoPago MetodoPago { get; set; }
        public List<DetalleFactura> DetallesFactura { get; set; }
    }
}

File: UNIDAD 1/00_SOA_SIN_BDD/002. REST_JAVA_CONUNI_GR10/02. CLIMOV/app/src/main/java/ec/edu/monster/modelo/Factura.kt
package ec.edu.monster.modelo

import java.util.*

data class Factura(
    val idFactura: Int,
    val numeroFactura: String,
    val fechaEmision: Date,
    val idEmpleado: Int,
    val idCliente: Int,
    val idMetodoPago: Int,
    val subtotal: Double,
    val descuento: Double,
    val iva: Double,
    val total: Double,
    val empleado: Empleado,
    val cliente: Cliente,
    val metodoPago: MetodoPago,
    val detallesFactura: List<DetalleFactura>
)
